assets/src/edit-story/components/fontPicker/fontPicker.js

```diff
@@ -135,7 +135,12 @@ function FontPicker({ onChange, lightMode = false, placeholder, value }) {
         <FontPickerTitle>{value || placeholder}</FontPickerTitle>
         <DropDownIcon />
       </FontPickerSelect>
-      <Popup anchor={ref} isOpen={isOpen} fillWidth={DEFAULT_WIDTH}>
+      <Popup
+        showOverflow
```

Comment:
Curious, what is this for?

Reply:
The TransitionWrapper on the video controls and elementLink frame also have this prop for their `Popup` component. It looks like the popup no longer is using this value.  I don't know the editor well enough to say with certainty that this can go while i do no it is not hurting anything. Happy to remove it, let me know. Carlos is on vacation but I can DM him and see if there's a specific reason to it.

Reply:
Looks like these should be removed from everywhere since this attribute was removed in #2306 
@wassgha -- assuming the `showOverflow` usage for `Popup` can be removed from the code everywhere and was left in accidentally, can you confirm?

Reply:
Correct!

Reply:
The showOverflow is to ensure the shadow shows up since the container used to clip it. If it doesn’t do that anymore then we can drop it.

Reply:
With `showOverflow`
`box-shadow: 0px 8px 10px rgba(0,0,0,0.15);` 
![Screen Shot 2020-06-30 at 9 30 22 AM](https://user-images.githubusercontent.com/10720454/86151967-57509580-bab4-11ea-9dbb-a331c4a025fd.png)

Without: 
`    box-shadow: 0px 8px 10px rgba(0,0,0,0.15);`
![Screen Shot 2020-06-30 at 9 33 43 AM](https://user-images.githubusercontent.com/10720454/86152321-db0a8200-bab4-11ea-8c17-d16ba689267f.png)


The box shadow on the popup component stays intact regardless of this prop so I am removing showOverflow from this update. 
I did not update the other areas to keep scope in check.